src/components/Blogs/index.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from "react";
+import SearchBar from "../SearchBar";
+import { Container } from "react-bootstrap";
+import { useTranslation } from "react-i18next";
+import "./index.scss";
+const NUM_BLOGS_TO_SHOW = 9;
+const BLOGS_URL =
+  "https://public-api.wordpress.com/wp/v2/sites/safeplace102505649.wordpress.com/posts";
+
+export default function Blogs() {
+  const { t } = useTranslation();
+  const [blogPosts, setBlogPosts] = useState([]);
+  const fetchBlogPosts = async () => {
```

Comment:
consider a newline above here too. it's like a "paragraph" in an essay. you're doing something new here so you separate it from the thing above.